uwblueprint/community-fridge-kw
# fix: homepage and verification auth icons

## Brief description. What is this change? 

### [Homepage visual enhancements and add icons](https://www.notion.so/uwblueprintexecs/Team-Hub-Phase-1-Enhancements-Phase-2-cf7744a5b04748059761c1649a61a463?p=ccd766234c614aa1964dbd94bd591bee)
### [Email verification expiry message](https://www.notion.so/uwblueprintexecs/Email-verification-expiry-message-e1898b9f7ef1410b8a44a2e3107e40ea)



## Implementation description. How did you make this change?
* Added numbers to desktop view

https://user-images.githubusercontent.com/53016529/151677730-d73192f7-50c8-493b-8cef-04d324a4080e.mov

* Added verification email images

https://user-images.githubusercontent.com/53016529/151677774-cef6e1a0-dfa7-4f9f-bd1f-9a6542405b7b.mov

[confirmed verification icon](https://drive.google.com/file/d/1710JqTk7lxUUBbvqG0S8Jqv_aiOPDL8i/view?usp=sharing)

* Added email verification expiry date

<img width="405" alt="Screen Shot 2022-01-31 at 11 47 14 AM" src="https://user-images.githubusercontent.com/53016529/151836229-9e6401ed-bbb7-40e4-90c9-f4d5acb8e3da.png">



## Steps to test
1. For donation step icons: test responsiveness with mobile/desktop and check it matches Figma
2. For verification email: check if it matches Figma by signing up for a new account
Note: ^ to test confirmed verification email page, change  in `App.tsx`, include `import Home from "./components/auth/Signup/ConfirmVerificationPage";`




## Checklist
